app/cmd/plugin_search.go

```diff
@@ -72,19 +72,70 @@ func searchPlugins(plugins *client.AvailablePluginList, keyword string) []client
 	return result
 }
 
+func matchPluginsData(plugins []client.AvailablePlugin) (result []client.CenterPlugin) {
+	result = make([]client.CenterPlugin, 0)
+	jclient := &client.UpdateCenterManager{
+		JenkinsCore: client.JenkinsCore{
+			RoundTripper: pluginSearchOption.RoundTripper,
+		},
+	}
+	getCurrentJenkinsAndClient(&(jclient.JenkinsCore))
+	site, err := jclient.GetSite()
+	if err != nil {
+		return
+	}
+	pluginJclient := &client.PluginManager{
+		JenkinsCore: client.JenkinsCore{
+			RoundTripper: pluginSearchOption.RoundTripper,
+		},
+	}
+	getCurrentJenkinsAndClient(&(pluginJclient.JenkinsCore))
+	plu, err := pluginJclient.GetPlugins()
+	if err != nil {
+		return
+	}
+
+	for _, plugin := range plugins {
+		for _, updatePlugin := range site.UpdatePlugins {
+			if plugin.Name == updatePlugin.Name {
+				result = append(result, updatePlugin)
+				break
```

Comment:
We already have the result here, `break` is not enough. Consider jumping the outside loop. https://github.com/jenkins-zh/jenkins-cli/pull/156/files#diff-a77a67bf776e10119612072fdd491d2fR98